docs/source/guide/get_started/installation.rst

```diff
@@ -53,24 +97,24 @@ OpenVINO™ Training Extensions command line functionality.
 Install OpenVINO™ Training Extensions for users (Intel GPUs)
 *************************************************************
 
-1. Install OpenVINO™ Training Extensions
-from source to use XPU functionality.
+1. Install OpenVINO™ Training Extensions from source to use Intel XPU functionality:
 
 .. code-block:: shell
 
-    # Clone the training_extensions repository with the following command:
     git clone https://github.com/open-edge-platform/training_extensions.git
     cd training_extensions
 
-    pip install -e '.[base]' --extra-index-url https://download.pytorch.org/whl/test/xpu
+    uv venv .otx --python 3.10 # or 3.11
+    source .otx/bin/activate
+
+    uv pip install -e '.[base]' --extra-index-url https://download.pytorch.org/whl/test/xpu
```

Comment:
Not an issue coming from this PR, but I don't think we should use test XPU torch package storage here